### PR TITLE
Issue#1312

### PR DESCRIPTION
**Pull request by upulindika**
Pull request title

- Clearly and concisely describes what it does
- Refer to the issue that it solves, if available

Pull request description

- Describes the main changes that come with the pull request
- Any relevant additional information is provided

> For detailed contributing instructions see https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

### CODE REVIEW

File: remote-facade/src/main/java/com/iluwatar/remotefacade/Order.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  /**
+   * Retrieve an entree order.
+   * 
+   * @return the order
+   */
+  public String getEntree() {
+    try {
+      ObjectInputStream inStream = new ObjectInputStream(this.socket.getInputStream());
+      String order = (String) inStream.readObject();
+      System.out.println("Retrieved order: \n" + order);
```
Comment (abimarank) [inline remote-facade/src/main/java/com/iluwatar/remotefacade/Order.java:118]: Care to remove `System.out` statements?

File: remote-facade/src/main/java/com/iluwatar/remotefacade/Order.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+   * @return the order
+   */
+  public String getSide() {
+    try {
+      ObjectInputStream inStream = new ObjectInputStream(this.socket.getInputStream());
+      String order = (String) inStream.readObject();
+      System.out.println("Retrieved order: \n" + order);
+      return order;
+    } catch (IOException | ClassNotFoundException e) {
+      e.printStackTrace();
```
Comment (abimarank) [inline remote-facade/src/main/java/com/iluwatar/remotefacade/Order.java:138]: Care to remove `e.printStackTrace();` statements?

**Review by abimarank — CHANGES_REQUESTED** [review]

**Review by iluwatar — CHANGES_REQUESTED** [review]
The build is failing due to old version number

**Comment by iluwatar** [comment]
Closed due to inactivity

**Pull request closed without merging**